common/src/main/java/org/conscrypt/OpenSSLCipherRSA.java

```diff
@@ -441,7 +441,9 @@ protected AlgorithmParameters engineGetParameters() {
                                 EvpMdRef.getJcaDigestAlgorithmStandardNameFromEVP_MD(mgf1Md)),
                         pSrc));
                 return params;
-            } catch (NoSuchAlgorithmException | InvalidParameterSpecException e) {
+            } catch (NoSuchAlgorithmException e) {
+                throw new RuntimeException("No providers of AlgorithmParameters.OAEP available");
```

Comment:
But we still need to catch the exception even if it's impossible, right? What is your suggestion here?